solang-parser/src/solidity.lalrpop

```diff
@@ -49,7 +57,8 @@ ImportRename: (Identifier, Option<Identifier>) = {
 
 PragmaDirective: SourceUnitPart = {
     // The lexer does special parsing for String literal; it isn't really a string literal
-    <l:@L> "pragma" <i:SolIdentifier> <s:StringLiteral> <r:@R> ";" => SourceUnitPart::PragmaDirective(Loc::File(file_no, l, r), i, s)
+    <l:@L> "pragma" <i:SolIdentifier> <s:StringLiteral> <r:@R> ";" => SourceUnitPart::PragmaDirective(Loc::File(file_no, l, r), Some(i), Some(s)) ,
+    <l:@L> "pragma" <false_token:!> <r:@R> ";" => {recovered_errs.push(false_token);     SourceUnitPart::PragmaDirective(Loc::File(file_no, l, r), None, None)} ,
```

Comment:
The formatting of the new lines in the file `solidity.lalrpop` is very bad. Please make it cleaner